about_us.html

```diff
@@ -50,67 +50,67 @@
                 <a class="menu-link" href="blog.html">BLOG</a>
                 <a class="menu-link" href="mailto:hello@ckcollab.com">CONTACT US</a>
             </div>
-            <div class="computer-container"></div>
-            <!--                <img class="computer" alt="computer.png" src="./assets/images/compy.svg">-->
-            <div id="carousel">
-                <v-carousel
-                        :height="computer_image_height"
-                        :width="computer_image_height"
-                        hide-delimiters
-                        :show-arrows="false"
-                        v-model="slide_counter">
-                    <v-carousel-item
-                            v-for="(slide, i) in slides"
-                            :key="i"
-                            :width="computer_image_height"
-                            :src="slide.src">
-                    </v-carousel-item>
-                </v-carousel>
-                <div class="arrow-container">
-                    <div @click="go_left" class="arrow left">
-                        <img class="left-arrow" alt="left-arrow.png" src="./assets/images/left%20key.svg">
+            <section class="content">
+                <div class="computer-container"></div>
+                <!--                <img class="computer" alt="computer.png" src="./assets/images/compy.svg">-->
+                <div id="carousel">
+                    <div class="left-arrow" @click="go_left">
+                        <img class="arrow-img" alt="left-arrow.png" src="./assets/images/left%20key.svg">
                     </div>
-                    <div @click="slide_counter++" class="arrow right">
-                        <img class="right-arrow" alt="right-arrow.png" src="./assets/images/right%20key.svg">
+                    <v-carousel
+                            height=""
```

Comment:
Do we need the height prop here or can we pull it?

Reply:
From what I could see, Vue automatically adds the height prop with the value of "500px". If you set it to an empty string then when it renders it has no prop and then uses the CSS.